### PR TITLE
From bot pr arm osx h58eea4

### PR DESCRIPTION
**Pull request by k-dominik**
this starts from #86 ...

### CODE REVIEW

**Comment by k-dominik** [comment]
@conda-forge-admin, please restart ci

**Comment by k-dominik** [comment]
hmm am I missing something, or why is CI not running?

**Comment by k-dominik** [comment]
okay @hmaarrfk, @stuarteberg, @jakirkham - given that CI passes I think this PR can be merged - and #86 can be closed.

**Comment by k-dominik** [comment]
Hmm okay I force pushed this morning with something that builds locally fine for me. I think the build errors on osx are not related...

```
dyld: Library not loaded: @rpath/libssl.3.dylib
  Referenced from: /Users/runner/miniforge3/conda-bld/vigra_1648108636439/_build_env/lib/libcurl.4.dylib
  Reason: image not found
/Users/runner/miniforge3/conda-bld/vigra_1648108636439/work/conda_build.sh: line 64:  8532 Abort trap: 6           cmake ${CMAKE_ARGS} .. -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_CXX_LINK_FLAGS="${LDFLAGS}" -DCMAKE_EXE_LINKER_FLAGS="${LDFLAGS}" -DCMAKE_CXX_FLAGS="${CXXFLAGS}" -DWITH_VIGRANUMPY=TRUE -DWITH_BOOST_THREAD=1 -DWITH_OPENEXR=1 -DWITH_LEMON=1 -DDEPENDENCY_SEARCH_PREFIX=${PREFIX} -DFFTW3F_INCLUDE_DIR=${PREFIX}/include -DFFTW3F_LIBRARY=${PREFIX}/lib/libfftw3f${SHLIB_EXT} -DFFTW3_INCLUDE_DIR=${PREFIX}/include -DFFTW3_LIBRARY=${PREFIX}/lib/libfftw3${SHLIB_EXT} -DHDF5_CORE_LIBRARY=${PREFIX}/lib/libhdf5${SHLIB_EXT} -DHDF5_HL_LIBRARY=${PREFIX}/lib/libhdf5_hl${SHLIB_EXT} -DHDF5_INCLUDE_DIR=${PREFIX}/include -DBoost_INCLUDE_DIR=${PREFIX}/include -DBoost_LIBRARY_DIRS=${PREFIX}/lib -DBoost_PYTHON_LIBRARY=${PREFIX}/lib/libboost_python${CONDA_PY}${SHLIB_EXT} -DPython_ROOT_DIR=${PREFIX} -DPython_FIND_VIRTUALENV=ONLY -DZLIB_INCLUDE_DIR=${PREFIX}/include -DZLIB_LIBRARY=${PREFIX}/lib/libz${SHLIB_EXT} -DPNG_LIBRARY=${PREFIX}/lib/libpng${SHLIB_EXT} -DPNG_PNG_INCLUDE_DIR=${PREFIX}/include -DTIFF_LIBRARY=${PREFIX}/lib/libtiff${SHLIB_EXT} -DTIFF_INCLUDE_DIR=${PREFIX}/include -DJPEG_INCLUDE_DIR=${PREFIX}/include -DJPEG_LIBRARY=${PREFIX}/lib/libjpeg${SHLIB_EXT} ${EXTRA_CMAKE_ARGS}
Traceback (most recent call last):
  File "/Users/runner/miniforge3/bin/conda-mambabuild", line 10, in <module>
    sys.exit(main())
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/boa/cli/mambabuild.py", line 239, in main
    call_conda_build(action, config)
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/boa/cli/mambabuild.py", line 211, in call_conda_build
    result = api.build(
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/conda_build/build.py", line 3088, in build_tree
    packages_from_this = build(metadata, stats,
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/conda_build/build.py", line 2211, in build
    utils.check_call_env(cmd, env=env, rewrite_stdout_env=rewrite_env,
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/conda_build/utils.py", line 410, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/Users/runner/miniforge3/lib/python3.9/site-packages/conda_build/utils.py", line 390, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/Users/runner/miniforge3/conda-bld/vigra_1648108636439/work/conda_build.sh']' returned non-zero exit status 134.
##[error]Bash exited with code '1'.

```

**Comment by k-dominik** [comment]
@conda-forge-admin, please restart ci

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
+build_platform:
+  osx_arm64: osx_64
+conda_forge_output_validation: true
```
Comment (jakirkham) [inline conda-forge.yml:3]: This key appears to be added twice (please see below), which looks like is causing the bot issues ( https://github.com/regro/cf-scripts/issues/1523 )

Comment (hmaarrfk) [reply conda-forge.yml:3]: Good find.

**Pull request merged**